src/System.Net.Http/pkg/System.Net.Http.pkgproj

```diff
@@ -1,42 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Yes.  It is part of NETStandard.Library.  The only thing the package was kept around for was to deliver the OOB impl to desktop.  Now that is part of the desktop support package.